src/Microsoft.Health.Fhir.Shared.Api/Controllers/ExportController.cs

```diff
@@ -111,17 +111,25 @@ public class ExportController : Controller
             [FromQuery(Name = KnownQueryParameterNames.Container)] string containerName,
             [FromQuery(Name = KnownQueryParameterNames.TypeFilter)] string typeFilter,
             [FromQuery(Name = KnownQueryParameterNames.Format)] string formatName,
+            [FromQuery(Name = KnownQueryParameterNames.AnonymizationConfigurationLocation)] string anonymizationConfigLocation,
+            [FromQuery(Name = KnownQueryParameterNames.AnonymizationConfigurationFileEtag)] string anonymizationConfigFileETag,
             string typeParameter)
         {
             CheckIfExportIsEnabled();
 
+            if (!string.IsNullOrWhiteSpace(anonymizationConfigFileETag) || !string.IsNullOrWhiteSpace(anonymizationConfigFileETag))
```

Comment:
>if (!string.IsNullOrWhiteSpace(anonymizationConfigFileETag) || !string.IsNullOrWhiteSpace(anonymizationConfigFileETag)) [](start = 12, length = 119)

Nit: I see three instances of this code in file. Can we make function like one above?

Reply:
Extract a unified ValidateForAnonymizedExport function. 
And I add a check to ensure that when only _anonymizationConfigEtag is given, will return bad request with "Missing '_anonymizationConfig' for anonymized export"